tests/mlio-py-test/test_data_reader_params.py

```diff
@@ -0,0 +1,44 @@
+import os
+import unittest
+
+import mlio
+from mlio.integ.numpy import as_numpy
+import numpy as np
+
+
+class TestDataReaderParams(unittest.TestCase):
```

Comment:
I would prefer using pytest instead of unittest.

Reply:
Okay, unittest removed.